requirements.txt

```diff
@@ -1,4 +1,6 @@
 Django>=4.0.1,<4.1.0
 djangorestframework>=3.13.1,<3.14.0
+django-filter>=21.1,<21.2
```

Comment:
In Python, `requirements.txt` files usually pin dependencies to specific versions instead of a range to ensure repeatable builds across environments and avoid dependency updates that can break production.

See:
- https://stackoverflow.com/questions/43658870/requirements-txt-vs-setup-py
- https://nvie.com/posts/pin-your-packages/

Please change this :)

Reply:
Got it! Great articles